Typings issue in version 1.9.0

If I run `tsc -d` on my Typescript source I get the following message:

```bash
../../../../../grpc/index.d.ts(69,50): error TS2314: Generic type 'Message<T>' requires 1 type argument(s).                                                  
```
grpc-node version: 1.9.0
Typescript version: 2.7.1
`tsconfig.json` compiler options:

```js
"compilerOptions": {
    "target": "es6",
    "module": "commonjs",
    "moduleResolution": "node",
    "emitDecoratorMetadata": true,
    "experimentalDecorators": true,
    "allowSyntheticDefaultImports": true
  }
```

--- a/packages/grpc-native-core/index.d.ts
+++ b/packages/grpc-native-core/index.d.ts
@@ -66,7 +66,7 @@ declare module "grpc" {
    * - Anything else becomes the relevant reflection object that ProtoBuf.js would create
    */
   export interface GrpcObject {
-    [name: string]: GrpcObject | typeof Client | Message;
+    [name: string]: GrpcObject | typeof Client | Message<any>;
   }
 
   /**

